code/exception/handler/abstract.php

```diff
@@ -354,6 +354,37 @@ public function isEnabled($type)
     }
 
     /**
+     * Exception Handler
+     *
+     * Do not call this method directly. Function visibility is public because set_exception_handler does not allow for
+     * protected method callbacks.
+     *
+     * @param  object $exception  The exception to be handled
+     * @return bool
+     */
+    public function _handleException($exception)
+    {
+        $result = false;
+
+        if($this->isEnabled(self::TYPE_EXCEPTION))
+        {
+            //Handle \Error Exceptions in PHP7
+            if (class_exists('Error') && $exception instanceof \Error)
+            {
+                $message = $exception->getMessage();
```

Comment:
How about, make rename `https://github.com/timble/kodekit/blob/master/code/exception/exception.php` to `Throwable` and only include if < PHP 7. Then implement this interface on the `ExceptionError`, and allow `ExceptionError` to act as a decorator, pass the PHP 7 exception in as the `$previous` argument http://php.net/manual/en/language.exceptions.extending.php, whereby you can then still call `getTrace()` and `getTraceAsString()` and that will proxy to the PHP 7 exception. Then you can transition your code to use the new `Throwable` interface (as renamed above), and then remove the decoration once that's done.